src/models/dashboard-items.js

```diff
@@ -19,7 +20,7 @@ export const dashboardItemsConfig = {
     },
     chartsDataCodes: {
         indicatorChart: ["RVC_SAFETY_BOXES", "RVC_ADS_WASTAGE", "RVC_DILUTION_SYRINGES_RATIO"],
-        coverageChart: ["RVC_CAMPAIGN_COVERAGE", "RVC_VACCINE_UTILITZATION"],
+        utilizationRateChart: ["RVC_VACCINE_UTILITZATION"],
```

Comment:
Typo

Reply:
@tokland this typo comes from the metadata, the indicator is saved with this typo

Reply:
Yup, fixed in vaccination-metadata in the PR we will be merging soon. Thx!